rest/client/rest.go

```diff
@@ -126,6 +125,77 @@ func (c *communicatorImpl) GetHosts(ctx context.Context, f func([]*model.APIHost
 	return nil
 }
 
+func (c *communicatorImpl) SetBannerMessage(ctx context.Context, m string) error {
+	info := requestInfo{
+		method:  post,
+		version: apiVersion2,
+		path:    "/admin/banner",
```

Comment:
I think these paths need to not have the leading /, otherwise there will be 2 in the url